Sources/UBFoundation/Networking/UBURLRequestModifier+TokenAuthorization.swift

```diff
@@ -30,3 +30,20 @@ public extension UBURLRequestTokenAuthorization {
         }
     }
 }
+
+/// A protocol describing a token authorisation request modifier
+@available(iOS 13.0, *)
+public protocol UBAsyncURLRequestTokenAuthorization: UBAsyncURLRequestModifier {
+    /// Fetches the token and returns it
+    ///
+    func getToken() async throws -> String
+}
+
+@available(iOS 13.0, *)
+public extension UBAsyncURLRequestTokenAuthorization {
```

Comment:
```suggestion
public extension UBAsyncURLRequestBearerTokenAuthorization {
```

Wenn nur bearer token benutzt werden, würde ich den Funktionsnamen explizit machen.

Reply:
können wir dann machen, wenn ubique mal was anderes macht ;-)